hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/request/volume/OMVolumeDeleteRequest.java

```diff
@@ -57,6 +60,36 @@ public OMVolumeDeleteRequest(OMRequest omRequest) {
     super(omRequest);
   }
 
+  @Override
+  public OMRequest preExecute(OzoneManager ozoneManager) throws IOException {
+    super.preExecute(ozoneManager);
+    DeleteVolumeRequest deleteVolumeRequest =
+        getOmRequest().getDeleteVolumeRequest();
+    Preconditions.checkNotNull(deleteVolumeRequest);
```

Comment:
nit: prefer JDK's `Objects.requireNonNull`

```suggestion
    Objects.requireNonNull(deleteVolumeRequest);
```